### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by willwray** [kookma/TW-Tamasha#17] [Q&A]
This looks fun!

I found the 'slider' plugin from tiddlywiki.com and started playing.
Then I saw mention of Tamasha in groups threads so came here.
This looks the more ambitious application.

I've a presentation to do mid-June and reckon I'll give Tamasha a spin.
Dev seems stalled - or going on outside github - so wondering about plans.

Can you say a couple of words on slider vs Tamasha,
and if there's an incoming release expected in the next month or so?

Thanks,  Will

**Comment by kookma** [comment]
Hi @willwray 
Thank you for your post! If you mean https://github.com/kookma/slider, it is a simple step trough tools to read a sequence of tiddler such as instructions and it is not for presentation purpose!

Tamasha instead targets the real stand alone presentation app based on web technology! It is a single page app on the shoulder of Tiddlywiki. I tried to resemble all features of reveal.js but for people with no or little web/css/js knowledge. There are other rivals like slidedev from @antfu which is highly powerful but its target audiences is developers.

The plan is to improve Tamasha to work on include (a rather long term roadmap)

- better font size handling
- support for author/viewer mode (I mean two screen one projected and one in front of author)
- simpler transition effects
- highlight portion of code when view a whole code
- simple theme selection/ more theme
- print to pdf
- ...

**Comment by kookma** [comment]
> Dev seems stalled - or going on outside github - so wondering about plans.

I am busy, but dev is not stopped! idea, comments, PR are welcome.
